components/library/libgnt/Makefile

```diff
@@ -0,0 +1,49 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL)". You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright (c) 2014 Alexander Pyhalov
+# Copyright 2017 Gary Mills
+# Copyright 2019 Michal Nowak
+#
+
+BUILD_BITS=64
+BUILD_STYLE=meson
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		libgnt
+COMPONENT_VERSION=	2.14.3
+COMPONENT_REVISION=	1
+COMPONENT_PROJECT_URL=	https://www.pidgin.im
```

Comment:
Please remove the COMPONENT_REVISION as this is a new package.

Reply:
It's also unlikely that I would have applied a copyright to the file in 2017.